content/en/docs/ops/ambient/upgrade/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Upgrade Guide
+description: Upgrade guide for Istio Ambient mesh.
+weight: 2
```

Comment:
This conflicts with install which is also at 2. I suspect we want this to be after that, so maybe 3 or more. But that will conflict with architecture which is currently 3, so need to bump the architecture page to a higher number as well.

Reply:
Right, I noticed this and then forgot to fix it.